src/controllers/review.js

```diff
@@ -0,0 +1,81 @@
+const dishModel = require('../models/dish');
+
+const CustomError = require('../utils/customError');
+
+const addReviewToDish = async (req, res, next) => {
+  const { dishId } = req.params;
+  const { rating, comment } = req.body;
+  const reviewData = { rating, comment };
+  try {
+    if (!rating) {
```

Comment:
what is this if for?

Reply:
const { rating, comment } = req.body; // get the rating and  review from customer req body
  const reviewData = { rating, comment }; // make  review item form this request 
  try {
    if (!rating) {  // check if rating is not null because it requred

Reply:
You should use express validator and the logic to middleware/validation.js